lib/runner/VirtualGoogleAssistantInvoker.js

```diff
@@ -96,14 +97,19 @@ class VirtualGoogleAssistantResponse extends InvokerResponse {
 
     prompt() {
         const richResponse = _.get(this.json, "data.google.richResponse.items[0].simpleResponse.textToSpeech");
+        const expressRichResponse = _.get(this.json, "payload.google.richResponse.items[0].simpleResponse.textToSpeech");
 
-        return richResponse || this.json.speech;
+        return expressRichResponse || richResponse || this.json.speech;
     }
 
     reprompt() {
         return undefined;
     }
 
+    supported() {
+        return true;
+    }
+
```

Comment:
you are supporting all kind of keys with this, are you sure about it?

Reply:
Yes, same as we do on Virtual Alexa